lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,154 @@
+    ], function (gettext, $, _, Backbone, AccountSettingsModel, AccountSettingsFieldViews, AccountSettingsView) {
+
+        var setupAccountSettingsSection = function (fields_data) {
+
+            var accountSettingsElement = $('.account-settings-container');
+            var accountSettingsModel = new AccountSettingsModel();
+            accountSettingsModel.url = accountSettingsElement.data('accounts-api-url');
+
+            var accountSettingsView = new AccountSettingsView({
+                el: accountSettingsElement,
+                model: accountSettingsModel,
+                context: {
+                    accountSettingsModelUrl: accountSettingsModel.url
+                },
+                sections: [
+                    {
+                        title: gettext("Basic Account Information"),
+                        fields: [
+                            {
+                                view: new AccountSettingsFieldViews.ReadonlyFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Username"),
+                                    valueAttribute: "username",
+                                    helpMessage: "",
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.TextFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Full Name"),
+                                    valueAttribute: "name",
+                                    helpMessage: gettext("This is used on your edX certificates, and all changes are reviewed."),
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.EmailFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Email"),
+                                    valueAttribute: "email",
+                                    helpMessage: gettext("You account email is used as a login mechanism and for course communications."),
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.PasswordFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Password"),
+                                    valueAttribute: "password",
+                                    emailAttribute: "email",
+                                    linkTitle: gettext("Reset Password"),
+                                    linkHref: fields_data['password']['url'],
+                                    helpMessage: gettext("To reset your password you'll need to click the reset password link sent to you via email."),
```

Comment:
Oh, OK. How about this:

When you click *Reset Password*, a message will be sent to your email address. Click the link in the message to reset your password.